.spin/bootstrap

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Running the scripts here helps Spin to do it in background
+# and then make a snapshot to save time at new instance creation.
+
+./bazel version
+
+# TODO: Make sure you have "/home/spin/.bazel_binaries/4.1.0/bin_t" in spin user path too.
+export PATH="/home/spin/.bazel_binaries/4.1.0/bin_t:$PATH"
+# Sets up go to definition in VS Code
+tools/scripts/build_compilation_db.sh
```

Comment:
These two lines depend on first compiling Sorbet in debug mode. The bazel version changes from time to time, so we need to export the correct version.

Also, building the compilation database requires Sorbet to be compiled first.

We would need something like this:
```shell
# Compile Sorbet
# This downloads the correct Bazel version and prints the version to the screen
./bazel build //main:sorbet --config=dbg

# After we build, we need to parse the output here to find out the correct version
BAZEL_VERSION=./bazel version # needs some output processing to get the right version

# Then we can export and build the compilation db
export PATH="/home/spin/.bazel_binaries/${BAZEL_VERSION}/bin_t:$PATH"

tools/scripts/build_compilation_db.sh
```